.travis.yml

```diff
@@ -4,6 +4,7 @@ node_js:
     - "0.12"
     - iojs
 sudo: false
+before_install: "if [[ `node --version` == *v0.10* ]]; then npm install -g npm; fi;"
```

Comment:
I added some [scoped packages](https://docs.npmjs.com/misc/scope) to the `devDependecnies` for testing. https://github.com/eslint/eslint/commit/ff066465fc55103c83dc89e68cee943d5f517b1a#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

[Node `0.10` build on Travis CI installs and uses npm v1.4.28](https://travis-ci.org/eslint/eslint/jobs/75478797#L93) which doesn't support scoped packages. Only npm >=2 is compatible with scoped packages. That is why I added a script to install the latest npm on `0.10` build.